### PR TITLE
[NAE-1723] EventLog classes are not resolved correctly after search in MongoDB collection 

### PR DESCRIPTION
**Pull request by machacjozef**
# Description

- fix history service

Fixes [NAE-1723]

### Third party dependencies

No new dependencies were introduced

### Blocking Pull requests

Depends on #110   // NAE-1684

## How Has Been This Tested?

manualy and test

- HistoryServiceTest  [HistoryServiceTest.groovy](https://github.com/netgrif/application-engine/pull/108/files#diff-c217ba2943f33b2b255fc88036b4bbe245bb9bc83b69fa76049d89c78c03b110)

### Test Configuration

| Name                | Tested on |
|---------------------| --------- |
| OS                  |   Linux Mint 20.3        |
| Runtime             |  Java 11        |
| Dependency Manager  |  Maven 3.8.4         |
| Framework version   |  Spring Boot 2.7.0     |
| Run parameters      |           |
| Other configuration |           |

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes have been checked, personally or remotely, with @mladoniczky 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have resolved all conflicts with the target branch of the PR
- [x] I have updated and synced my code with the target branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes:
    - [x] Lint test
    - [x] Unit tests
    - [x] Integration tests
- [x] I have checked my contribution with code analysis tools:
    - [x] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_application-engine)
    - [x] [Snyk](https://app.snyk.io/org/netgrif/project/ea82b3b8-658d-41f5-89a7-76cd600da01f)
- [x] I have made corresponding changes to the documentation:
    - [x] Developer documentation
    - [x] User Guides
    - [x] Migration Guides

[NAE-1723]: https://netgrif.atlassian.net/browse/NAE-1723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/java/com/netgrif/application/engine/petrinet/domain/Place.java [hunk 1]
```diff
@@ -44,6 +44,7 @@ public Place clone() {
         clone.setTokens(this.tokens);
         clone.setIsStatic(this.isStatic);
         clone.setTitle(this.getTitle());
+        clone.setPosition(this.getPosition().getX(), this.getPosition().getY());
```
Comment (renczesstefan) [inline src/main/java/com/netgrif/application/engine/petrinet/domain/Place.java:47]: `.getPosition()` may be null?

File: src/main/java/com/netgrif/application/engine/petrinet/web/responsebodies/PlaceImportReference.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+@Data
+public class PlaceImportReference extends Node {
+
+    private Integer tokens;
+
+    private Boolean isStatic;
+
+    public PlaceImportReference(Place place) {
+        this.setPosition(place.getPosition().getX(), place.getPosition().getY());
```
Comment (renczesstefan) [inline src/main/java/com/netgrif/application/engine/petrinet/web/responsebodies/PlaceImportReference.java:15]: `.getPosition()` may be null?

File: src/main/java/com/netgrif/application/engine/petrinet/web/responsebodies/TransitionImportReference.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+import com.netgrif.application.engine.petrinet.domain.Node;
+import com.netgrif.application.engine.petrinet.domain.Transition;
+import lombok.Data;
+
+@Data
+public class TransitionImportReference extends Node {
+
+    public TransitionImportReference(Transition transition) {
+        this.setPosition(transition.getPosition().getX(), transition.getPosition().getY());
```
Comment (renczesstefan) [inline src/main/java/com/netgrif/application/engine/petrinet/web/responsebodies/TransitionImportReference.java:11]: Same as above.

File: src/main/java/com/netgrif/application/engine/petrinet/domain/Place.java (hunk 1 above)
Comment (machacjozef) [reply src/main/java/com/netgrif/application/engine/petrinet/domain/Place.java:47]: This from  PR #110

**Review by mazarijuraj — APPROVED** [review]

**Pull request merged**